be/src/vec/exec/format/orc/vorc_reader.h

```diff
@@ -177,6 +177,12 @@ class OrcReader : public GenericReader {
     Status get_parsed_schema(std::vector<std::string>* col_names,
                              std::vector<TypeDescriptor>* col_types) override;
 
+    Status get_parsed_col_name_iceberg_ids(std::vector<std::string>* col_names,
```

Comment:
Other components like `tvf` will get the file schema/metadata, How to provide a public interface to get the schema/metadata. `get_parsed_col_name_iceberg_ids` may be correct, but it is not elegant enough.